src/Locale/nl/cake.po

```diff
@@ -1,297 +1,312 @@
-msgid "on {0}"
-msgstr "op {0}"
+#: vendor/cakephp/cakephp/src/I18n/RelativeTimeFormatter.php:371
+msgid "on %s"
+msgstr "op %s"
 
-#: I18n/RelativeTimeFormatter.php:47;126;316
+#: vendor/cakephp/cakephp/src/I18n/RelativeTimeFormatter.php:47;126;316
 msgid "{0} year"
 msgid_plural "{0} years"
 msgstr[0] "{0} jaar"
 msgstr[1] "{0} jaren"
 
-#: I18n/RelativeTimeFormatter.php:51;129;319
+#: vendor/cakephp/cakephp/src/I18n/RelativeTimeFormatter.php:51;129;319
 msgid "{0} month"
 msgid_plural "{0} months"
 msgstr[0] "{0} maand"
 msgstr[1] "{0} maanden"
 
-#: I18n/RelativeTimeFormatter.php:57;132;322
+#: vendor/cakephp/cakephp/src/I18n/RelativeTimeFormatter.php:57;132;322
 msgid "{0} week"
 msgid_plural "{0} weeks"
 msgstr[0] "{0} week"
 msgstr[1] "{0} weken"
 
-#: I18n/RelativeTimeFormatter.php:59;135;325
+#: vendor/cakephp/cakephp/src/I18n/RelativeTimeFormatter.php:59;135;325
 msgid "{0} day"
 msgid_plural "{0} days"
 msgstr[0] "{0} dag"
 msgstr[1] "{0} dagen"
 
-#: I18n/RelativeTimeFormatter.php:64;138
+#: vendor/cakephp/cakephp/src/I18n/RelativeTimeFormatter.php:64;138
 msgid "{0} hour"
 msgid_plural "{0} hours"
 msgstr[0] "{0} uur"
 msgstr[1] "{0} uren"
 
-#: I18n/RelativeTimeFormatter.php:68;141
+#: vendor/cakephp/cakephp/src/I18n/RelativeTimeFormatter.php:68;141
 msgid "{0} minute"
 msgid_plural "{0} minutes"
 msgstr[0] "{0} minuut"
 msgstr[1] "{0} minuten"
 
-#: I18n/RelativeTimeFormatter.php:72;144
+#: vendor/cakephp/cakephp/src/I18n/RelativeTimeFormatter.php:72;144
 msgid "{0} second"
 msgid_plural "{0} seconds"
 msgstr[0] "{0} seconde"
 msgstr[1] "{0} seconden"
 
-#: Network/Response.php:1472
-msgid "The requested file was not found"
-msgstr "Het gevraagde bestand werd niet gevonden"
-
-#: ORM/RulesChecker.php:57
+#: vendor/cakephp/cakephp/src/ORM/RulesChecker.php:57
 msgid "This value is already in use"
 msgstr "Deze waarde is al in gebruik"
 
-#: ORM/RulesChecker.php:104
+#: vendor/cakephp/cakephp/src/ORM/RulesChecker.php:104
 msgid "This value does not exist"
 msgstr "Deze waarde bestaat niet"
 
-#: ORM/RulesChecker.php:128
+#: vendor/cakephp/cakephp/src/ORM/RulesChecker.php:128
 msgid "The count does not match {0}{1}"
 msgstr "Het aantal komt niet overeen {0}{1}"
 
-#: Utility/Text.php:894
+#: vendor/cakephp/cakephp/src/Utility/Text.php:902
 msgid "and"
 msgstr "en"
 
-#: Validation/Validator.php:103
+#: vendor/cakephp/cakephp/src/Validation/Validator.php:111
 msgid "This field is required"
 msgstr "Dit veld is verplicht"
 
-#: Validation/Validator.php:104
+#: vendor/cakephp/cakephp/src/Validation/Validator.php:112
 msgid "This field cannot be left empty"
 msgstr "Dit veld mag niet leeg zijn"
 
-#: Validation/Validator.php:1712
+#: vendor/cakephp/cakephp/src/Validation/Validator.php:1885
 msgid "The provided value is invalid"
 msgstr "De opgegeven waarde is niet geldig"
 
-#: View/Helper/FormHelper.php:963
-msgid "New {0}"
-msgstr "Nieuw {0}"
+#: vendor/cakephp/cakephp/src/View/Helper/FormHelper.php:1039
+msgid "New %s"
```

Comment:
Arent we using {0} now etc?

Reply:
We should but cakephp/cakephp core does not in this case.
I ran i18n extract and updated the po file by the pot file (like you should, using PoEdit).

Reply:
I see, those two are used in `$legend = sprintf($actionName, $modelName);` :)
Tricky ones..